efficientdet/keras/efficientdet_arch_keras_test.py

```diff
@@ -51,31 +51,32 @@ def test_model_output(self):
     with tf.Session(graph=tf.Graph()) as sess:
       feats = tf.ones(inputs_shape)
       tf.random.set_random_seed(SEED)
-      feats = efficientdet_arch_keras.build_backbone(feats, config)
-      feats = efficientdet_arch_keras.build_feature_network(feats, config)
-      feats = efficientdet_arch_keras.build_class_and_box_outputs(feats, config)
-      # TODO(tanmingxing): Fix the failure for keras Model.
-      # feats = efficientdet_arch_keras.EfficientDetModel(config=config)(feats)
+      # feats = efficientdet_arch_keras.build_backbone(feats, config)
+      # feats = efficientdet_arch_keras.build_feature_network(feats, config)
+      # v = efficientdet_arch_keras.build_class_and_box_outputs(feats, config)
+      v = efficientdet_arch_keras.EfficientDetModel(config=config)(feats)
       sess.run(tf.global_variables_initializer())
-      keras_class_out, keras_box_out = sess.run(feats)
+      keras_class_out, keras_box_out = sess.run(v)
     with tf.Session(graph=tf.Graph()) as sess:
       feats = tf.ones(inputs_shape)
       tf.random.set_random_seed(SEED)
       feats = legacy_arch.efficientdet(feats, config=config)
       sess.run(tf.global_variables_initializer())
       legacy_class_out, legacy_box_out = sess.run(feats)
     for i in range(3, 8):
-      self.assertAllEqual(keras_class_out[i - 3], legacy_class_out[i])
-      self.assertAllEqual(keras_box_out[i - 3], legacy_box_out[i])
+      self.assertAllClose(
+          keras_class_out[i - 3], legacy_class_out[i], rtol=1e-4, atol=1e-4)
```

Comment:
@fsx950223 
With rtol and ato,  EfficientDetModel can close results in graph mode, but the difference in eager mode is still big (line 78/79).